README.md

```diff
@@ -0,0 +1,239 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+- **[React js]**
+
+- **[Redux toolkit]**
+
+- **[CSS/bootstrap]**
+
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+
+- **[React props usage]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+- [live demo link](https://meteo-forecast.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+
+```sh
+ install vs code on computer
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+
+
+```sh
+  cd `<folder name>`
+  git clone https://github.com/ijosue16/metrics-app.git
+```
+-
+
+### Install
+
+Install this project with:
+
+
+
+```sh
+  Open folder using vs code
+
+  open terminal 
+
+  run "npm install"
+
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+
+
+```sh
+
+run " npm run build" and then
+
+run " npm start"
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+
+
+
+```sh
+  run "npm run test"
+```
+
+
+### Deployment
+
+You can deploy this project using:
+
+Github pages or Render.
+
+
+<!-- ``` Github pages
+
+``` -->
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Iradukunda Josue**
+
+- GitHub: [@ijosue16](https://github.com/ijosue16)
+<!-- - Twitter: [@iradukunda_23](https://twitter.com/iradukunda_23) -->
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+<!-- 👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+<!-- > Describe 1 - 3 features you will add to the project. -->
+
+- **Add Edit functionality**
+
+- **Add update progress functionality**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+
+If you like this project offer me a star ⭐
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Microverse team.
+
+I would like to thank Microverse team
```

Comment:
- [ ] Kindly ensure your README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).